static/app/views/issueList/issueViews/issueViewsList/issueViewsList.tsx

```diff
@@ -62,6 +76,45 @@ function IssueViewSection({createdBy, limit, cursorQueryParam}: IssueViewSection
   );
 }
 
+function SortDropdown() {
+  const navigate = useNavigate();
+  const location = useLocation();
+  const sort = useIssueViewSort();
+
+  return (
+    <CompactSelect
+      value={sort}
+      triggerProps={{
+        icon: <IconSort />,
+      }}
+      onChange={newSort => {
+        navigate({
+          pathname: location.pathname,
+          query: {sort: newSort.value},
+        });
+      }}
+      options={[
```

Comment:
do we want any analytics on this options changing/being selected

Reply:
Ah you're probably right, will add